VERSIONING.md

```diff
@@ -1,31 +1,58 @@
+marked it as stable or it is not specific to OpenTelemetry and is marked stable
+from by some other means (for example the GDI specification).
+
+As a concrete example, the 1.0 Splunk Distribution of OpenTelemetry Java
+release will apply to the tracing signal and the tracing components
+OpenTelemetry marked stable in its 1.0 release as well as the components marked
+as stable in the GDI specification 1.0 release. Some non-exhaustive examples of
+components that are not stable in the 1.0 release, meaning breaking changes may
+be introduced, include OpenTelemetry semantic conventions, automatic
+instrumentation libraries, and the metric signal.
 
 ## Version numbers
 
-The project strictly follows [Semantic Versioning 2.0.0](https://semver.org/). This means
+The project follows [Semantic Versioning 2.0.0](https://semver.org/). This means
 that all artifacts have a version of the format `MAJOR.MINOR.PATCH`.
-No backwards-incompatible changes will be made unless incrementing the `MAJOR` version number. 
-Most releases will be made by incrementing the `MINOR` version. 
-Patch releases will be made by incrementing the `PATCH` version.
 
-## Compatibility  
+- No backwards-incompatible changes will be made unless incrementing the `MAJOR` version number.
+- Most releases will be made by incrementing the `MINOR` version.
+- Patch releases will be made by incrementing the `PATCH` version.
 
-Backwards-incompatible changes will be avoided - if they are necessary, the `MAJOR` version of the artifact will be
-incremented.
+The version number is not kept in sync with the OpenTelemetry Java project,
+however the [changelog](CHANGELOG.md) will make it clear what version of
+OpenTelemetry Java a version is based against.
 
-This can occur in following situations:
-- OpenTelemetry Java Instrumentation breaking change - please see [upstream repository](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/VERSIONING.md) for their versioning policy
-- change in a Splunk-managed configuration property or code
+## Compatibility
 
-In the latter case, we will first **deprecate** (at least one release earlier) before introducing the change, so that customers can prepare accordingly.
+For stable components, backwards-incompatible changes will be avoided. If they
+are necessary, the `MAJOR` version of the artifact will be incremented.
+
+This can occur in situations including:
+
+- OpenTelemetry Java Instrumentation breaking change - please see [upstream
+  repository](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/VERSIONING.md)
+  for their versioning policy
+- Change in a Splunk-managed configuration property or code
+
+In the latter case, we will first **deprecate** (at least one release earlier)
+before introducing the change, so that customers can prepare accordingly.
 
 All the changes are to be communicated in the [changelog](CHANGELOG.md).
```

Comment:
Should we explicitly mention that unstable features (like metrics) MAY be changed in a backwards incompatible way between `MINOR` releases?